### ISSUE TITLE
Build Failure

### ISSUE THREAD

**Issue by alunduil** [gentoo/gentoo-docker-images#29]
Not sure which merge started this or if it's due to the base image but here's the latest build status on stage3-amd64-nomultilib from dockerhub:

```
lient:
 Version:      1.8.1
 API version:  1.20
 Go version:   go1.4.2
 Git commit:   d12ea79
 Built:        Thu Aug 13 02:49:29 UTC 2015
 OS/Arch:      linux/amd64

Server:
 Version:      1.8.3-rc4
 API version:  1.20
 Go version:   go1.4.2
 Git commit:   9f1606a
 Built:        Fri Oct  2 23:36:50 UTC 2015
 OS/Arch:      linux/amd64
Step 0 : FROM busybox
c51f86c28340 Verifying Checksum
c51f86c28340 Download complete
039b63dd2cba Verifying Checksum
039b63dd2cba Download complete
039b63dd2cba Extracting
039b63dd2cba Extracting
039b63dd2cba Extracting
039b63dd2cba Extracting
039b63dd2cba Extracting
039b63dd2cba Extracting
039b63dd2cba Extracting
039b63dd2cba Extracting
039b63dd2cba Extracting
039b63dd2cba Extracting
039b63dd2cba Extracting
039b63dd2cba Extracting
039b63dd2cba Extracting
039b63dd2cba Extracting
039b63dd2cba Extracting
039b63dd2cba Extracting
039b63dd2cba Extracting
039b63dd2cba Extracting
039b63dd2cba Extracting
039b63dd2cba Extracting
039b63dd2cba Extracting
039b63dd2cba Extracting
039b63dd2cba Extracting
039b63dd2cba Pull complete
c51f86c28340 Extracting
c51f86c28340 Extracting
c51f86c28340 Extracting
c51f86c28340 Pull complete
Digest: sha256:eb3c0d4680f9213ee5f348ea6d39489a1f85a318a2ae09e012c426f78252a6d2
Status: Downloaded newer image for busybox:latest
---> c51f86c28340
Step 1 : MAINTAINER Gentoo Docker Team
---> bf3be088470c
Step 2 : ADD build.sh /
---> ffa6009644f5
Step 3 : RUN /build.sh amd64 x86_64 -nomultilib
Downloading and extracting 20151203/stage3-amd64-nomultilib-20151203.tar.bz2...
[91mwget: can't execute 'ssl_helper': No such file or directory
[0m
[91mwget: error getting response: Connection reset by peer
[0m
[91mchmod: /busybox: No such file or directory
[0m
[91m/build.sh: line 14[0m
[91m: /busybox: not found
[0m
[91m/build.sh[0m
[91m: [0m
[91mline [0m
[91m15[0m
[91m: [0m
[91m/busybox[0m
[91m: [0m
[91mnot found[0m
[91m
[0m
[91m/build.sh[0m
[91m: [0m
[91mline [0m
[91m16[0m
[91m: [0m
[91m/busybox[0m
[91m: [0m
[91mnot found[0m
[91m
[0m
[91m/build.sh[0m
[91m: [0m
[91mline [0m
[91m17[0m
[91m: [0m
[91m/busybox[0m
[91m: [0m
[91mnot found[0m
[91m
[0m
[91m/build.sh[0m
[91m: [0m
[91mline [0m
[91m20[0m
[91m: [0m
[91m/busybox[0m
[91m: [0m
[91mnot found[0m
[91m
[0m
Bootstrapped  into /:
total 0
drwxr-xr-x    1 root     root         134 Dec  6 05:03 [1;34m.[0m
drwxr-xr-x    1 root     root         134 Dec  6 05:03 [1;34m..[0m
-rwxr-xr-x    1 root     root           0 Dec  6 05:02 [1;32m.dockerenv[0m
-rwxr-xr-x    1 root     root           0 Dec  6 05:02 [1;32m.dockerinit[0m
drwxr-xr-x    1 root     root        4.2K Oct 31 17:14 [1;34mbin[0m
drwxr-xr-x    5 root     root         360 Dec  6 05:02 [1;34mdev[0m
drwxr-xr-x    1 root     root          90 Dec  6 05:02 [1;34metc[0m
drwxr-xr-x    1 root     root           6 Oct 31 17:15 [1;34mhome[0m
drwxr-xr-x    1 root     root         120 Dec  6 05:03 [1;34mnewWorldOrder[0m
dr-xr-xr-x  117 root     root           0 Dec  6 05:02 [1;34mproc[0m
drwxr-xr-x    1 root     root           0 Oct 31 17:15 [1;34mroot[0m
dr-xr-xr-x   13 root     root           0 Dec  6 05:02 [1;34msys[0m
drwxrwxrwt    1 root     root           0 Oct 31 17:15 [1;34mtmp[0m
drwxr-xr-x    1 root     root           8 Oct 31 17:15 [1;34musr[0m
drwxr-xr-x    1 root     root          16 Oct 31 17:15 [1;34mvar[0m
---> 10d26e747baa
Step 4 : RUN echo "default" > /run/openrc/softlevel
[91m/bin/sh[0m
[91m: can't create /run/openrc/softlevel: nonexistent directory
[0m
```

**Comment by alunduil** [comment]
This is also happening on stage3-amd64:

```
Client:
 Version:      1.8.1
 API version:  1.20
 Go version:   go1.4.2
 Git commit:   d12ea79
 Built:        Thu Aug 13 02:49:29 UTC 2015
 OS/Arch:      linux/amd64

Server:
 Version:      1.8.3-rc4
 API version:  1.20
 Go version:   go1.4.2
 Git commit:   9f1606a
 Built:        Fri Oct  2 23:36:50 UTC 2015
 OS/Arch:      linux/amd64
Step 0 : FROM busybox
ac6a7980c6c2 Verifying Checksum
ac6a7980c6c2 Download complete
c00ef186408b Verifying Checksum
c00ef186408b Download complete
c00ef186408b Extracting
c00ef186408b Extracting
c00ef186408b Extracting
c00ef186408b Extracting
c00ef186408b Extracting
c00ef186408b Extracting
c00ef186408b Extracting
c00ef186408b Extracting
c00ef186408b Extracting
c00ef186408b Extracting
c00ef186408b Extracting
c00ef186408b Extracting
c00ef186408b Extracting
c00ef186408b Extracting
c00ef186408b Extracting
c00ef186408b Extracting
c00ef186408b Extracting
c00ef186408b Extracting
c00ef186408b Extracting
c00ef186408b Extracting
c00ef186408b Extracting
c00ef186408b Extracting
c00ef186408b Extracting
c00ef186408b Pull complete
ac6a7980c6c2 Extracting
ac6a7980c6c2 Extracting
ac6a7980c6c2 Extracting
ac6a7980c6c2 Pull complete
Digest: sha256:e4f93f6ed15a0cdd342f5aae387886fba0ab98af0a102da6276eaf24d6e6ade0
Status: Downloaded newer image for busybox:latest
---> ac6a7980c6c2
Step 1 : MAINTAINER Gentoo Docker Team
---> 3c141d3549fe
Step 2 : ADD build.sh /
---> 77c0922d4f48
Step 3 : RUN /build.sh amd64 x86_64
Downloading and extracting 20151217/stage3-amd64-20151217.tar.bz2...
[91mwget: can't execute 'ssl_helper': No such file or directory
[0m
[91mwget: error getting response: Connection reset by peer
[0m
[91mchmod: /busybox: No such file or directory
[0m
[91m/build.sh[0m
[91m: [0m
[91mline [0m
[91m14[0m
[91m: [0m
[91m/busybox[0m
[91m: [0m
[91mnot found[0m
[91m
[0m
[91m/build.sh[0m
[91m: [0m
[91mline [0m
[91m15[0m
[91m: [0m
[91m/busybox[0m
[91m: [0m
[91mnot found[0m
[91m
[0m
[91m/build.sh[0m
[91m: [0m
[91mline [0m
[91m16[0m
[91m: [0m
[91m/busybox[0m
[91m: [0m
[91mnot found[0m
[91m
[0m
[91m/build.sh[0m
[91m: [0m
[91mline [0m
[91m17[0m
[91m: [0m
[91m/busybox[0m
[91m: [0m
[91mnot found[0m
[91m
[0m
[91m/build.sh[0m
[91m: [0m
[91mline [0m
[91m20[0m
[91m: [0m
[91m/busybox[0m
[91m: [0m
[91mnot found[0m
[91m
[0m
Bootstrapped  into /:
total 0
drwxr-xr-x    1 root     root         134 Dec 23 05:46 [1;34m.[0m
drwxr-xr-x    1 root     root         134 Dec 23 05:46 [1;34m..[0m
-rwxr-xr-x    1 root     root           0 Dec 23 05:45 [1;32m.dockerenv[0m
-rwxr-xr-x    1 root     root           0 Dec 23 05:45 [1;32m.dockerinit[0m
drwxr-xr-x    1 root     root        4.2K Dec  8 16:44 [1;34mbin[0m
drwxr-xr-x    5 root     root         360 Dec 23 05:45 [1;34mdev[0m
drwxr-xr-x    1 root     root          90 Dec 23 05:45 [1;34metc[0m
drwxr-xr-x    1 nobody   nogroup        0 Dec  8 16:44 [1;34mhome[0m
drwxr-xr-x    1 root     root         130 Dec 23 05:46 [1;34mnewWorldOrder[0m
dr-xr-xr-x  117 root     root           0 Dec 23 05:45 [1;34mproc[0m
drwxr-xr-x    1 root     root           0 Dec  8 16:44 [1;34mroot[0m
dr-xr-xr-x   13 root     root           0 Dec 23 05:45 [1;34msys[0m
drwxrwxrwt    1 root     root           0 Dec  8 16:44 [1;34mtmp[0m
drwxr-xr-x    1 root     root           8 Dec  8 16:44 [1;34musr[0m
drwxr-xr-x    1 root     root          16 Dec  8 16:44 [1;34mvar[0m
---> 75b6cf56a149
Step 4 : RUN echo "default" > /run/openrc/softlevel
[91m/bin/sh: can't create /run/openrc/softlevel[0m
[91m: nonexistent directory
[0m
```

**Comment by ChaosEngine** [comment]
I think it is related to wgetting busybox from https://busybox.net/.
Looks like the web page enforces HTTPS and do auto-redirect from HTTP to HTTPS. While building  container, inernal wget does not have SSL installed (not compiled with option probably) and fails with this `can't execute 'ssl_helper'`

**Comment by hasufell** [comment]
I have this too in my [paludis image](https://github.com/hasufell/docker-gentoo-amd64-paludis). I can confirm that it works after applying https://github.com/gentoo/gentoo-docker-images/pull/27

**Comment by globalcitizen** [comment]
For anyone waiting for a fix I suggest using [lxc-gentoo](https://github.com/globalcitizen/lxc-gentoo) instead. Just delete the `lxc.conf` output and you're good to go on docker. Plus, you get cryptographic validation of sources, caching for re-runs, etc.

**Comment by alunduil** [comment]
Will let these run now that #27 has been merged and see if that fixes it.  Will close tomorrow if everything is successful.

**Comment by alunduil** [comment]
Fixed with merge of #27.
